internal/worker/remoterelations/remoteapplicationworker.go

```diff
@@ -72,9 +82,9 @@ type relation struct {
 	localDead      bool
 	suspended      bool
 	localUnitCount int
-	localRuw       *relationUnitsWorker
-	remoteRuw      *relationUnitsWorker
-	remoteRrw      *remoteRelationsWorker
+	localRuw       ReportableWorker
+	remoteRuw      ReportableWorker
+	remoteRrw      ReportableWorker
 
 	applicationToken   string // token for app in local model
```

Comment:
Just call these `localApplicationToken` and `localRelationToken` and delete the comments.